app/src/main/proto/vss.proto

```diff
@@ -0,0 +1,153 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.vss;
+
+message GetObjectRequest {
+
+  //  StoreId: All APIs operate within a single storeId.
```

Comment:
Feel like this sentence should go after the next one and doesn't need to be prefixed by "StoreId:".